frontend/src/employee-frontend/components/messages/MessagesPage.tsx

```diff
@@ -133,14 +137,15 @@ export default React.memo(function MessagesPage() {
           <MessageEditor
             message={message}
             onChange={(message: Message) => setMessage(message)}
-            onClose={() => setShowEditor(false)}
+            onClose={closeEditor}
             onSend={() =>
-              postMessage(accounts.value[0].id, {
+              view?.account &&
+              postMessage(view.account.id, {
```

Comment:
Shouldn't the `MessageEditor` give the selected sender account in the callback instead of deducing it from the selected account in that page state?
It would feel more natural if the editor was initialized with the selected account. It's needed there anyway because we want to at least show the sender (if not even select it).

<img width="313" alt="newmsg" src="https://user-images.githubusercontent.com/1176169/119116645-1c8b7f00-ba31-11eb-948e-d369711dcc5c.png">